### PR TITLE
CAL-1317:intermitent exception after starting second cluster node

### PR DESCRIPTION
**Pull request by azaoui**
PR description : In cluster mode when there are running calendar jobs executing some queries on index while others node are starting and index is offline for syncro reasons , this exception is occurred .
The fix will detect index offline by  catching the IndexOfflineRepositoryException and we just log the error when trace is enable

### CODE REVIEW

File: calendar-service/src/main/java/org/exoplatform/calendar/service/PopupReminderJob.java [hunk 1]
```diff
@@ -126,6 +127,10 @@ public void run() {
             }
           }
         }
+      } catch (IndexOfflineRepositoryException e) {
+        if (log_.isTraceEnabled()) {
+          log_.trace("An exception occurred: " + e.getMessage());
```
Comment (thomasdelhomenie) [inline calendar-service/src/main/java/org/exoplatform/calendar/service/PopupReminderJob.java]: Can you put a more meaningful message (something like "Error while running Calendar PopupReminderJob job:") and add the exception in the trace (do not swallow the exception) ?

File: calendar-service/src/main/java/org/exoplatform/calendar/service/ReminderJob.java [hunk 2]
```diff
@@ -165,6 +167,10 @@ public void run() {
             }
           }
         }
+      } catch (IndexOfflineRepositoryException e) {
+        if (log_.isTraceEnabled()) {
+          log_.trace("An exception occurred: " + e.getMessage());
```
Comment (thomasdelhomenie) [inline calendar-service/src/main/java/org/exoplatform/calendar/service/ReminderJob.java]: Can you put a more meaningful message (something like "Error while running Calendar ReminderJob job:") and add the exception in the trace (do not swallow the exception) ?

Comment (azaoui) [reply calendar-service/src/main/java/org/exoplatform/calendar/service/ReminderJob.java]: done

File: calendar-service/src/main/java/org/exoplatform/calendar/service/PopupReminderJob.java (hunk 1 above)
Comment (azaoui) [reply calendar-service/src/main/java/org/exoplatform/calendar/service/PopupReminderJob.java]: done

File: calendar-service/src/main/java/org/exoplatform/calendar/service/PopupReminderJob.java [hunk 3]
```diff
@@ -25,18 +25,19 @@
...
 import org.exoplatform.container.ExoContainerContext;
 import org.exoplatform.container.component.ComponentRequestLifecycle;
 import org.exoplatform.job.MultiTenancyJob;
 import org.exoplatform.services.jcr.ext.common.SessionProvider;
+import org.exoplatform.services.jcr.impl.core.query.lucene.IndexOfflineRepositoryException;
 import org.exoplatform.services.log.ExoLogger;
 import org.exoplatform.services.log.Log;
 import org.exoplatform.services.organization.OrganizationService;
 import org.exoplatform.ws.frameworks.cometd.ContinuationService;
+import org.quartz.JobExecutionContext;
```
Comment (thomasdelhomenie) [inline calendar-service/src/main/java/org/exoplatform/calendar/service/PopupReminderJob.java]: Useless import.

File: calendar-service/src/main/java/org/exoplatform/calendar/service/PopupReminderJob.java (hunk 1 above)
Comment (thomasdelhomenie) [reply calendar-service/src/main/java/org/exoplatform/calendar/service/PopupReminderJob.java]: The exception is still swallowed.

File: calendar-service/src/main/java/org/exoplatform/calendar/service/ReminderJob.java (hunk 2 above)
Comment (thomasdelhomenie) [reply calendar-service/src/main/java/org/exoplatform/calendar/service/ReminderJob.java]: The exception is still swallowed.

**Pull request merged**